### PR TITLE
[tests-only][full-ci]Refactor expected-failure-api-on-ocis-file

### PR DESCRIPTION
**Pull request by amrita-shrestha**
## Description

This PR refactor  expected failure file related to the closed issue 

## Related Issue

- https://github.com/owncloud/core/pull/40392

## How Has This Been Tested?

- locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: .drone.env [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # The test runner source for API tests
-CORE_COMMITID=31529dcebe93e1d856d7034a8c0bbbde3b782390
-CORE_BRANCH=master
+CORE_COMMITID=a8daac90c27d27c85e4798396af45ae97737576d
+CORE_BRANCH=refactorExpectedFailureClosedIssue
```
Comment (SwikritiT) [inline .drone.env:3]: Just a remainder to update these to master

**Review by grgprarup — COMMENTED** [review]
LGTM :+1:

**Review by SagarGi — COMMENTED** [review]
lgtm :+1:

**Review by individual-it — APPROVED** [review]

**Comment by SwikritiT** [comment]
@amrita-shrestha Just a reminder that these changes might need to be updated in Reva as well. Could you please make a PR for `master` and `edge` and mention them on this issue as well https://github.com/owncloud/QA/issues/767. Thanks!

**Pull request merged**